src/Services/BaseService.php

```diff
@@ -284,8 +284,8 @@ public function selectHelper($sqlUpToFromStatement, $map)
     /**
      * Build and Throw Invalid Value Exception
      *
-     * @param $message              - The error message which will be displayed
-     * @param $type                 - Type of Exception
+     * @param $message - The error message which will be displayed
+     * @param $type    - Type of Exception
```

Comment:
We need to keep an eye on to ensure proper syntax e.g.
* @param $message - The error message which will be displayed

should be
* @param string $message - The error message which will be displayed

etc.

Reply:
I will say in my defense... haha.  Good catch.
![image](https://user-images.githubusercontent.com/228117/129720244-7663a5dd-fb9f-45c9-9e05-ff42e1cb2c85.png)